app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,12 @@
+html {
+  box-sizing: border-box;
```

Comment:
Whoops, I believe this is set up wrong. This style is correct but we're missing the inheritance.

https://www.paulirish.com/2012/box-sizing-border-box-ftw/